readthedocs/extra/advanced-usage/sessions.rst

```diff
@@ -44,3 +44,53 @@ methods. For example, you could save it on a database:
 
 You should read the ````session.py```` source file to know what "relevant
 data" you need to keep track of.
+
+
+Sessions and Heroku
+-------------------
+
+You probably have a newer version of SQLite installed (>= 3.8.2). Heroku uses
+SQLite 3.7.9 which does not support `WITHOUT ROWID`. So, If you generated your
+session file on a system with SQLite >= 3.8.2 your session file will not work
+on Heroku's platform and will throw a corrupted schema error.
+
+There are multiple ways to solve this, the easiest of which is generating a
+session file on your Heroku dyno itself. The most complicated is creating
+a custom buildpack to install SQLite >= 3.8.2.
+
+
+Generating Session File on a Heroku Dyno
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Do not restart your application Dyno at any point prior to retrieving your
```

Comment:
You could use a ``.. note::` or even `.. warning::` block. Makes more sense than all bold (see other files for usage examples).